src/routes/tasks.js

```diff
@@ -75,7 +76,7 @@ router.post("/", (request, response) => {
  * */
 router.get("/:id", (request, response) => {
     Tasks.findById(request.params.id)
-        .then((resp) => response.status(200).json(resp))
+        .then((resp) => response.status(200).json({ data: resp, error: null }))
```

Comment:
this is how the "data" key is added to the top level of the return data. Pretty slick, eh?